From 06ece089d16e5e12af94db372f95882214b5a9ad Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Fri, 8 Jan 2016 19:03:17 +0100
Subject: [PATCH] Fix Debian FTBS compilation error.

---
 lib/lirc_options.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/lib/lirc_options.c
+++ b/lib/lirc_options.c
@@ -84,8 +84,9 @@
 static char* parse_O_arg(int argc, char** argv)
 {
 	char* path = NULL;
+	int i;
 
-	for (int i = 0; i < argc; i += 1) {
+	for (i = 0; i < argc; i += 1) {
 		if (strcmp(argv[i], "-O") != 0 &&
 		    strcmp(argv[i], "--options-file") != 0)
 			continue;
