From 9759ff9a33f10ccbd7915851ed9cb542cb99fe17 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Wed, 31 May 2017 10:11:10 +0200
Subject: [PATCH 4/7] doc: Deprecate lirc.hwdb, NEWS update.

---
 NEWS          | 21 ++++++++++++---------
 doc/data2hwdb |  6 ++++--
 2 files changed, 16 insertions(+), 11 deletions(-)

diff --git a/NEWS b/NEWS
index 45a1b97..0976715 100644
--- a/NEWS
+++ b/NEWS
@@ -1,4 +1,8 @@
 0.10.0rc1 28/5/17
+ * ftdi driver: Fix off-by-2 timings (#275). This is a breaking
+   change, see ftdi driver docs or source at doc/plugins/ftdi.html.
+ * doc: The lirc.hwdb parsable driver list is deprecated - use
+   the python database or the configs/*.yaml files instead.
  * lircd: the deprecated --uinput option is dropped. Use
    the lircd-uinput(8) daemon instead.
  * lircd: The --release option is deprecated.
@@ -6,31 +10,30 @@
    the hardware directly on start and exits on errors.
  * lircd, lircd-uinput: Change default release suffix due to linux
    kernel changes (#263).
- * The slinke (a. k. a. slink-e) driver is dropped.
+ * plugins: The slinke (a. k. a. slink-e) driver is dropped.
  * plugins: Add support for irdroid in the irtoy driver.
  * plugins: Enhanced device enumeration, mode2 --list-devices
    works for most drivers.
+ * plugins: devinput driver: Fix bad codelength (#263).
  * New, shiny (i. e., unstable) python API. See the API docs,
    module "python bindings". The API is also available as a pypi
    package, typically at /usr/share/lirc/python-pkg
  * lircd-uinput: Multiple config file parsing errors (#284).
- * The configs/*conf files no more supports  enumerating devices
-   using shell scripts.
- * Added enumeration support and server version level check to
-   driver API.
+ * lib: Added enumeration support and server version level check
+   to driver API.
  * irrecord: Add --driver-options for lircd symmetry (#277).
  * lircd: Fix devices not connected at first attempt (#279).
  * lib: Fix failure when parsing absolute paths config files (#281).
  * lircd/irsend: Fix bug when sending/receiving concurrently for
    some remotes (#278, 5 pages!)
  * lircd:Fix missing systemd support due to upstream changes (#274).
+ * The configs/*conf files no more supports  enumerating devices
+   using shell scripts.
  * Build: Added new configure --enable-devinput option (#264).
  * Build: Added new configure --enable-uinput option, ditched
-   HAVE_UINPUT 0.9.4c cludge.
+   the HAVE_UINPUT 0.9.4c kludge.
  * Build: More quiet, enhanced V=0/V=1 support.
- * ftdi driver: Fix off-by-2 timings (#275).
- * devinput driver: Fix bad codelength (#263).
- * OSX patches(#273)
+ * Build: OSX patches(#273)
 
 0.9.4d 23/01/17
  * The debian packaging means that freebsd and hurd builds are
diff --git a/doc/data2hwdb b/doc/data2hwdb
index 5db9bc8..fea307c 100644
--- a/doc/data2hwdb
+++ b/doc/data2hwdb
@@ -10,9 +10,11 @@ import database
 HEADER = '''
 # LIRC - Hardware DataBase
 #
-# THIS IS A GENERATED FILE. DO NOT EDIT.
+# THIS FILE IS DEPRECATED AND WILL BE REMOVED IN NEXT RELEASE.
 #
-# This file lists all the remote controls supported by LIRC
+# This is a generated file. Do not edit.
+#
+# The file lists all the remote controls supported by LIRC
 # in a parseable form. It's legacy file kept for compatiblity.
 # The current yaml files configs/*.conf should be a better
 # source in most cases.
-- 
2.9.3

