From c1defe015831c0a8dbd09d24ae003b513eb7828a Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Wed, 31 May 2017 10:54:52 +0200
Subject: [PATCH 5/7] doc: Mute annoying build error message.

---
 doc/Makefile.am | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 310761b..7336a45 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -148,13 +148,15 @@ docpage.xsl: $(abs_builddir)/driver-toc.xsl
 $(srcdir)/manpage.xsl: $(abs_builddir)/driver-toc.xsl
 
 table.html: docpage.xsl
-	test -e lirc || ln -s $(top_srcdir)/tools/lirc-setup lirc || :
+	test -e lirc || \
+	    ln -s $(top_srcdir)/tools/lirc-setup lirc 2>/dev/null || :
 	PYTHONPATH=$(top_srcdir)/python-pkg/lirc \
 	$(PYTHON) $(srcdir)/data2table $(top_srcdir)/configs ../configs \
 	        | xsltproc --html docpage.xsl - > table.html
 
 lirc.hwdb:
-	test -e lirc || ln -s $(top_srcdir)/tools/lirc-setup lirc || :
+	test -e lirc || \
+	    ln -s $(top_srcdir)/tools/lirc-setup lirc  2>/dev/null || :
 	PYTHONPATH=$(top_srcdir)/python-pkg/lirc \
 	$(PYTHON) $(srcdir)/data2hwdb $(top_srcdir)/configs ../configs \
 	    > lirc.hwdb
-- 
2.9.3

