From 8cb5fd91935603536dcb0377ff91eb0ae477d19a Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Thu, 18 Aug 2016 16:19:17 +0200
Subject: [PATCH 05/10] contrib: disable udev /dev/uinput rule.

---
 contrib/97-lircd-uinput.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/97-lircd-uinput.rules b/contrib/97-lircd-uinput.rules
index 12228a7..a554299 100644
--- a/contrib/97-lircd-uinput.rules
+++ b/contrib/97-lircd-uinput.rules
@@ -7,4 +7,4 @@ SUBSYSTEMS=="input",ATTRS{name}=="lircmd",SYMLINK+="lircmd"
 # Make /dev/uinput accessible for the lirc group. A setfacl RUN would be
 # better here, but seemingly does not work (?)
 # An alternative is to use lircd-setup(8) to sun setfacl(1).
-KERNEL=="uinput",GROUP:="lirc",MODE:="0660"
+## KERNEL=="uinput",GROUP:="lirc",MODE:="0660"
-- 
2.5.5

