From aa087cb999b7303d6f14d0928a84909d5bf29529 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Thu, 18 Aug 2016 21:04:07 +0200
Subject: [PATCH 07/10] contrib: Fix usb devices' acl permissions (rhbz
 #1364744).

---
 contrib/60-lirc.rules | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/contrib/60-lirc.rules b/contrib/60-lirc.rules
index 8147546..492aae9 100644
--- a/contrib/60-lirc.rules
+++ b/contrib/60-lirc.rules
@@ -1,8 +1,9 @@
-# Installing this udev rule will make the /dev/lirc* devices accessible
-# for users in the group "lirc" using regular group permissions. USB
-# devices grants rw permission to the lirc group using ACL.
-# To enable, copy to /etc/udev/rules.d.
+# Make the /dev/lirc* devices accessible for users in the group "lirc"
+# using regular group permissions.
 
 KERNEL=="lirc[0-9]*", SUBSYSTEM=="lirc", GROUP="lirc", MODE="0660"
-ACTION=="add", SUBSYSTEM=="usb", \
-   RUN+="/usr/bin/setfacl -m g:lirc:rw $env{DEVNAME}"
+
+# Grant rw permission to the lirc group for USB devices using acl(5).
+
+ACTION=="add", SUBSYSTEM=="usb", ENV{DEVNAME}=="?*", \
+    RUN+="/usr/bin/sh -c '/usr/bin/setfacl -m g:lirc:rw %E{DEVNAME} 2>/dev/null || :'"
-- 
2.5.5

