From 63bb3aefffb4995156123dd6d29471597cd2be80 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Wed, 31 May 2017 20:32:00 +0200
Subject: [PATCH 7/7] Build: fix linking issue.

---
 lib/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/lib/Makefile.am b/lib/Makefile.am
index 50d1fce..db76b26 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -10,8 +10,6 @@ BUILT_SOURCES               = lirc/input_map.inc
 AM_CPPFLAGS                 = -I$(top_srcdir) -Wall -Wp,-D_FORTIFY_SOURCE=2 \
                               @kernel_header_flags@
 
-AM_LDFLAGS                  = @LIBUSB_LIBS@
-
 lib_LTLIBRARIES             = liblirc.la liblirc_client.la liblirc_driver.la \
                               libirrecord.la
 
@@ -45,6 +43,7 @@ liblirc_client_la_SOURCES   = lirc_client.c\
 			      lirc_log.h
 
 liblirc_driver_la_LDFLAGS   = -version-info 0:0:0
+liblirc_driver_la_LIBADD    = liblirc.la $(LIBUSB_LIBS)
 liblirc_driver_la_SOURCES   = driver.h \
                               drv_enum.c \
                               drv_enum.h \
-- 
2.9.3

