--- a/lirc-driver.pc
+++ b/lirc-driver.pc
@@ -10,7 +10,7 @@
 pkgdatadir=${prefix}/share/lirc
 docdir=${prefix}/share/doc/${PACKAGE_TARNAME}
 
-Libs: -llirc_driver
+Libs: -L${libdir} -llirc_driver
 Cflags: -I${includedir} -fpic -DPLUGINDOCS='"${docdir}/plugindocs"'
 
 plugindir=${libdir}/lirc/plugins
--- a/lirc.pc
+++ b/lirc.pc
@@ -10,5 +10,5 @@
 pkgdatadir=${datarootdir}/lirc
 plugindir=${libdir}/lirc/plugins
 
-Libs: -llirc
+Libs: -L${libdir} -llirc
 Cflags: -I${includedir}
