From eb06327fc1b4e8e9682cf471643ccbab35fed241 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Sat, 29 Oct 2016 11:50:23 +0200
Subject: [PATCH] systemd: Fix wrong path to irexec (#239).

---
 systemd/irexec.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: lirc-0.9.4c/systemd/irexec.service
===================================================================
--- lirc-0.9.4c.orig/systemd/irexec.service	2016-10-30 13:35:42.000000000 +0100
+++ lirc-0.9.4c/systemd/irexec.service	2016-10-30 13:35:53.000000000 +0100
@@ -20,7 +20,7 @@
 
 
 Type=simple
-ExecStart=/usr/sbin/irexec /etc/lirc/irexec.lircrc
+ExecStart=/usr/bin/irexec /etc/lirc/irexec.lircrc
 
 [Install]
 WantedBy=multi-user.target
