From 761c3c3f035192610cd563747ed7e3c205fbf937 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Wed, 16 Nov 2016 08:03:13 +0100
Subject: [PATCH] systemd: Add missing lircd-setup.service to distribution
 (#243).

---
 systemd/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/systemd/Makefile.am b/systemd/Makefile.am
index bfa339d..1cd80b5 100644
--- a/systemd/Makefile.am
+++ b/systemd/Makefile.am
@@ -2,6 +2,7 @@
 ACLOCAL_AMFLAGS             = -I m4
 
 dist_systemdsystemunit_DATA = lircd.socket lircd.service \
+                              lircd-setup.service \
                               lircd-uinput.service \
                               lircmd.service \
                               irexec.service
-- 
2.5.5

