Index: lirc-0.9.4c/lib/lirc_client.c
===================================================================
--- lirc-0.9.4c.orig/lib/lirc_client.c
+++ lirc-0.9.4c/lib/lirc_client.c
@@ -40,6 +40,10 @@

 #include "lirc_client.h"

+#ifndef MAXPATHLEN
+#define MAXPATHLEN 4096
+#endif
+
 /** Timeout in lirc_read_string. */
 static const struct timeval CMD_TIMEOUT = { .tv_sec = 1, .tv_usec = 0 };

Index: lirc-0.9.4c/plugins/atilibusb.c
===================================================================
--- lirc-0.9.4c.orig/plugins/atilibusb.c
+++ lirc-0.9.4c/plugins/atilibusb.c
@@ -34,6 +34,10 @@

 #include "lirc_driver.h"

+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
 #define CODE_BYTES 5
 #define USB_TIMEOUT (1000 * 60)

Index: lirc-0.9.4c/plugins/awlibusb.c
===================================================================
--- lirc-0.9.4c.orig/plugins/awlibusb.c
+++ lirc-0.9.4c/plugins/awlibusb.c
@@ -43,6 +43,10 @@

 #include "lirc_driver.h"

+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
 #define AW_MODE_LIRCCODE 1

 #define AWUSB_RECEIVE_BYTES 5
Index: lirc-0.9.4c/plugins/dfclibusb.c
===================================================================
--- lirc-0.9.4c.orig/plugins/dfclibusb.c
+++ lirc-0.9.4c/plugins/dfclibusb.c
@@ -35,6 +35,10 @@

 #include "lirc_driver.h"

+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
 #define CODE_BYTES 6
 #define USB_TIMEOUT (5000)

Index: lirc-0.9.4c/tools/irpty.cpp
===================================================================
--- lirc-0.9.4c.orig/tools/irpty.cpp
+++ lirc-0.9.4c/tools/irpty.cpp
@@ -142,6 +142,10 @@ static void copy_loop(int ptym, int igno
 	lirc_deinit();
 }

+#undef RESET
+#undef RAW
+#undef CBREAK
+
 static struct termios save_termios;
 static int ttysavefd = -1;
 static enum {
--- lirc-0.9.4c.orig/include/media/lirc.h.original	2016-12-26 16:34:04.000000000 +0000
+++ lirc-0.9.4c/include/media/lirc.h	2016-12-26 16:34:09.000000000 +0000
@@ -18,6 +18,9 @@
 #include <linux/ioctl.h>
 #else
 #include <sys/ioctl.h>
+#if defined(_IOT_SIMPLE) && !defined(_IOT__IOTBASE___u32)
+#define _IOT__IOTBASE___u32 _IOT_SIMPLE(__u32)
+#endif
 #endif

 #define PULSE_BIT       0x01000000

