From 2454802b4e493c07d8faa49374f12bebf29d7cd8 Mon Sep 17 00:00:00 2001
From: Alec leamas <al@mageia.localdomain>
Date: Tue, 30 May 2017 11:34:32 +0200
Subject: [PATCH] build: mageia fixes

---
 configure.ac | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 3d3c4c9..512ed71 100644
--- a/configure.ac
+++ b/configure.ac
@@ -135,9 +135,8 @@ enable_uinput_default="no"
 test -e "/dev/uinput" && enable_uinput_default="yes"
 
 version_nodots=$( $PYTHON << EOF
-v = "$VERSION".split("rc")[[0]].split("-")[[0]].split(".")
-v = dict(enumerate(v))
-print(int(v.get(0)) * 10000 + int(v.get(1)) * 100 + int(v.get(2)))
+v = "$VERSION".split("rc").pop(0).split("-").pop(0).split(".")
+print(int(v.pop(0)) * 10000 + int(v.pop(0)) * 100 + int(v.pop(0)))
 EOF
 )
 AC_DEFINE_UNQUOTED(VERSION_NODOTS, [$version_nodots])

