From 6b8fad0fdfd026d437ab6b953e701f2ece307e45 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Thu, 18 Aug 2016 21:04:07 +0200
Subject: [PATCH] contrib: Fix usb devices' acl permissions (rhbz #1364744).

---
 contrib/60-lirc.rules | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/contrib/60-lirc.rules b/contrib/60-lirc.rules
index 8147546..687009b 100644
--- a/contrib/60-lirc.rules
+++ b/contrib/60-lirc.rules
@@ -1,8 +1,9 @@
-# Installing this udev rule will make the /dev/lirc* devices accessible
-# for users in the group "lirc" using regular group permissions. USB
-# devices grants rw permission to the lirc group using ACL.
-# To enable, copy to /etc/udev/rules.d.
+# Make the /dev/lirc* devices accessible for users in the group "lirc"
+# using regular group permissions.
 
 KERNEL=="lirc[0-9]*", SUBSYSTEM=="lirc", GROUP="lirc", MODE="0660"
-ACTION=="add", SUBSYSTEM=="usb", \
+
+# Grant rw permission to the lirc group for USB devices using acl(5).
+
+ACTION=="add", SUBSYSTEM=="usb", ENV{DEVNAME}=="?*", \
    RUN+="/usr/bin/setfacl -m g:lirc:rw $env{DEVNAME}"
-- 
2.5.5

