From d310e7986c1fa062f7d08c41b4782e68e932576e Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Wed, 31 May 2017 11:05:32 +0200
Subject: [PATCH] Build: Use correct python interpreter.

---
 Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 7c57c5b..17a2ed0 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -194,7 +194,8 @@ devinput.lircd.conf:
 	echo "# Rename to devinput.lircd.dist if not using devinput driver" > $@
 	echo "# Re-generate for current kernel using lirc-make-devinput" >> $@
 	echo '#' >> $@
-	$(srcdir)/tools/lirc-make-devinput $(DEVINPUT_HEADER) >> $@
+	PYTHON=$(PYTHON) \
+	    $(srcdir)/tools/lirc-make-devinput $(DEVINPUT_HEADER) >> $@
 
 .phony:
 
-- 
2.9.3

